src/content/en/2020/fonts.md

```diff
@@ -276,34 +317,50 @@ Lots more information about color fonts, including examples, are available at [c
 
 ## Variable fonts
 
+{{ figure_markup(
+  caption="Usage of variable fonts on mobile.",
+  content="11.00%",
+  classes="big-number",
+  sheets_gid="91894176",
+  sql_file="04_14a.variable_font_comparison_fcp.sql"
+) }}
+
+Variable fonts are certainly one of the biggest stories this year. They're seen in 10.54% of desktop pages, and 11.00% of mobile. That's up from an average of 1.8% last year, a huge growth factor. It is not hard to see why their popularity is increasing – they offer more design flexibility, and also potentially smaller binary font sizes, especially if multiple styles of the same font are used on the same page.
+
+{# That's huge usage! Any ideas what's driven it? It would appear no variable font has any where near that usage so how come total usage is so high? Has any particular service or platform turned this on by default? #}
+
+{# TODO - authors Should we add a paragraph or bullet list explaing the common font axis and what they mean before launching into their usage? What is difference between wght and wdth? And between slnt and ital? Web Almanac readers may come here from pother chapters and not be familiar with these terms. Plus I am a little familair and don't know these answers! #}
```

Comment:
@raphlinus / @jpamental thoughts?

Reply:
I'm always keen to add more education about that, though I wonder if it might be better to introduce it lightly and just direct them here: https://variablefonts.io/about-variable-fonts/
(I explain all the axes and have little working examples of each)

Reply:
That works for me. Can you replace this comment text with a short paragraph about that? Feel free to either submit directly to this branch or just use the "Insert a suggestion" button above to replace this line.